cmd/agentflow/basic_scraper.flow

```diff
@@ -0,0 +1,39 @@
+metrics scraper "default"  {
+    targets = [
+        {
+            targets = [{ "__address__" = "localhost:9001"}]
```

Comment:
This assumes you have avalanche going `docker run -p 9001:9001 quay.io/freshtracks.io/avalanche --metric-count=1`